cardano-node/src/Cardano/Node/Tracing/Tracers/P2P.hs

```diff
@@ -593,6 +634,18 @@ instance MetaTrace (TracePeerSelection SockAddr) where
       Namespace [] ["ChurnMode"]
     namespaceFor TraceKnownInboundConnection {} =
       Namespace [] ["KnownInboundConnection"]
+    namespaceFor TraceLedgerStateJudgementChanged {} =
+      Namespace [] ["LedgerStateJudgementChanged"]
+    namespaceFor TraceOnlyBootstrapPeers {} =
+      Namespace [] ["OnlyBootstrapPeers"]
+    namespaceFor TraceUseBootstrapPeersChanged {} =
+      Namespace [] ["UseBootstrapPeersChanged"]
+    namespaceFor TraceBootstrapPeersFlagChangedWhilstInSensitiveState =
+      Namespace [] ["BootstrapPeersFlagChangedWhilstInSensitiveState"]
+    namespaceFor TraceOutboundGovernorCriticalFailure {} =
+      Namespace [] ["OutboundGovernorCriticalFailure"]
+    namespaceFor TraceDebugState {} =
+      Namespace [] ["DebugState"]
```

Comment:
Those new messages seem to lack entries for `documentFor`, `severityFor` and  `allNamespaces` in `instance MetaTrace (TracePeerSelection SockAddr)`.